base/db/R/assign.treatments.R

```diff
@@ -0,0 +1,48 @@
+#-------------------------------------------------------------------------------
+# Copyright (c) 2012 University of Illinois, NCSA.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the
+# University of Illinois/NCSA Open Source License
+# which accompanies this distribution, and is available at
+# http://opensource.ncsa.illinois.edu/license.html
+#-------------------------------------------------------------------------------
+
+##-----------------------------------------------------------------------------#
+##' Change treatments to sequential integers
+##'
+##' Assigns all control treatments the same value, then assigns unique treatments
+##' within each site. Each site is required to have a control treatment.
+##' The algorithm (incorrectly) assumes that each site has a unique set of experimental
+##' treatments.
```

Comment:
```suggestion
##' treatments. This assumption is required by the data in BETTdb that does not always consistently name treatments or quantity them in the managements table. Also it avoids having the need to estimate treatment by site interactions in the meta analysis model. This model is  uses data in the control treatment to estimate model parameters so the impact of the assumption is minimal.
```

Could someone please check this? It is late and I'd appreciate another set of eyes. I just cringed at the previous version that I probably wrote long ago.

Reply:
I fixed a typo in your suggestion and added it to the commit - it looks ok to me and we can always go back and change it when reviewing the functions in further detail. Again, this PR was really only to reorganize.